lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -109,25 +109,17 @@ def get_threads(request, course_key, discussion_id=None, per_page=THREADS_PER_PA
     #if the user requested a group explicitly, give them that group, otherwise, if mod, show all, else if student, use cohort
 
     group_id = request.GET.get('group_id')
-    is_cohorted = is_commentable_cohorted(course_key, discussion_id)
 
-    if group_id in ("all", "None"):
+    if group_id == "all":
         group_id = None
 
-    if not cached_has_permission(request.user, "see_all_cohorts", course_key):
-        group_id = get_cohort_id(request.user, course_key)
-        if not group_id and get_cohorted_threads_privacy(course_key) == 'cohort-only':
-            default_query_params['exclude_groups'] = True
+    if not group_id:
+        if not cached_has_permission(request.user, "see_all_cohorts", course_key):
```

Comment:
It looks like this permissions check will now not happen when a group_id is explicitly specified. Is there some other check in place to prevent a malicious user from requesting the threads of a group_id they are not supposed to have permission to view?